components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -7174,14 +7130,14 @@ public void deletePolicyGroup(String applicationId, String policyGroupId) throws
                 try {
                     conn.rollback();
                 } catch (SQLException e1) {
-                    log.error("Rollback while deleting the policy group", e);
+                    log.error("Rollback while deleting the policy group : " + policyGroupId, e);
                 }
             }
-			String strDataContext =
-					"(applicationId:" + applicationId + ", policyGroupId:" + policyGroupId + ")";
-			handleException("Error while executing the query to delete XACML policies : " + query + " : " +
-					strDataContext + " : " + e.getMessage(), e);
-		} finally {
+            String strDataContext =
```

Comment:
This line break automatically get applied when the code is formatted with wso2 formatter